fast follow for SL docs (#2288)
fishtown-analytics/docs.getdbt.com: website/docs/docs/use-dbt-semantic-layer/quickstart-semantic-layer.md

:::info 📌

> The dbt Semantic Layer is currently available in Public Preview for multi-tenant dbt Cloud accounts hosted in North America (logging in via https://cloud.getdbt.com/)

Reviewer: ```suggestion
The dbt Semantic Layer is currently available in Public Preview for multi-tenant dbt Cloud accounts hosted in North America. You can access the Semantic Layer when you log into [dbt Cloud](https://cloud.getdbt.com).
```

## Set up dbt Semantic Layer

Before continuing, you must have a multi-tenant dbt Cloud account hosted in North America to set up the dbt Semantic Layer in dbt Cloud. 

>   * Team and Enterprise accounts will be able to set up the Semantic Layer and [Metadata API](/docs/dbt-cloud-apis/metadata-api) in the integrated

Reviewer: Can we change these bullets to the present tense?  For example:
```suggestion
  * **Team and Enterprise accounts:** You can set up the Semantic Layer and [Metadata API](/docs/dbt-cloud-apis/metadata-api) in the integrated
```

    partner tool to import metric definition. 
  * Developer accounts will be able to query the Proxy Server using SQL but won't be able to browse dbt metrics in external tools, which requires access
    to the Metadata API.

To query your dbt metrics in an integrated partner tool, you need to [set up the dbt Semantic Layer](/docs/use-dbt-semantic-layer/setup-dbt-semantic-layer#set-up-dbt-semantic-layer) in dbt Cloud to connect with your integration tool:

1. In your dbt Cloud account, go to **Account Settings** and then **Service Tokens** to create a new [service account API token](docs/dbt-cloud-apis/service-tokens). 

>     * Permissions for service account tokens depend on your integration tool of choice. Review the [integration partners documentation](https://www.getdbt.com/product/semantic-layer-integrations) to clarify what permission sets you need.

Reviewer 2: 👍

>     * Permissions for service account tokens depend on your integration tool of choice. Review the [integration partners documentation](https://www.getdbt.com/product/semantic-layer-integrations) to clarify what permission sets you need.

Reviewer: ```suggestion
    * Assign permissions to service account tokens depending on the integration tool you choose. You can review the [integration partner documentation](https://www.getdbt.com/product/semantic-layer-integrations) to determine the permission sets you need to assign.
```

Reviewer: Consider making this bullet into a new step if you are doing this action now. Are you assigning permissions at this time?

Reviewer: A few options to consider to streamline these steps and prevent one place being changed and the other forgotten:
* Consider linking to these steps over here: https://deploy-preview-2288--docs-getdbt-com.netlify.app/docs/use-dbt-semantic-layer/setup-dbt-semantic-layer#set-up-dbt-semantic-layer
* Possibly consider making these steps a reusable so we can have one source in both places.

    <div>The dbt Semantic Layer does not store, or cache, or log your data. On each query to the Semantic Layer, the resulting data passes through dbt Cloud servers where it is never stored, cached, or logged. The data from your data platform gets routed through dbt Cloud servers, to your connecting data tool.</div>
  </div>
</details>
<details>
    <summary>Is the dbt Semantic Layer open source?</summary>
  <div>
    <div>Some components of the dbt Semantic Layer are open source like dbt-core, the dbt_metrics package, and the BSL licensed dbt-server. The dbt Proxy Server (what is actually compiling the dbt code) and the Metadata API are not open source. <br></br><br></br>

During Public Preview, the dbt Semantic Layer is open to all dbt Cloud tiers (Developer, Team, and Enterprise).<br></br><br></br>
<ul>    

> <li>dbt Core CLI users can define metrics in their dbt Core projects and calculate them using macros from the metrics package. To use the dbt Semantic Layer integrations, you will need to have a dbt Cloud account.</li><br></br><br></br>

Reviewer 3: Same as above

> <li>dbt Core CLI users can define metrics in their dbt Core projects and calculate them using macros from the metrics package. To use the dbt Semantic Layer integrations, you will need to have a dbt Cloud account.</li><br></br><br></br>

Author: ```suggestion
<li>dbt Core users can define metrics in their dbt Core projects and calculate them using macros from the metrics package. To use the dbt Semantic Layer integrations, you will need to have a dbt Cloud account.</li><br></br><br></br>
```

  <div>

>     <div>Metrics cannot be defined on <a href="https://docs.getdbt.com/docs/build/materializations#ephemeral">ephemeral models</a> because we reference the underlying table in the query that generates the metric so we need the table/view to exist in the database. If your table/view does not exist in your database, you'll likely see this error message:

Reviewer: ```suggestion
    <div>Metrics cannot be defined on <a href="https://docs.getdbt.com/docs/build/materializations#ephemeral">ephemeral models</a> because we reference the underlying table in the query that generates the metric so we need the table/view to exist in the database. If your table/view does not exist in your database, you might see this error message:
```

>  <code>Object 'DATABASE.SCHEMA.TESTING_EPHEMERAL does not exist or not authorized.</code><br></br>

Reviewer 3: Can we change the name here? Only realizing right now that it is an error message from me testing something. We could call it DATABASE.SCHEMA.METRIC_MODEL_TABLE?

>  <code>Object 'DATABASE.SCHEMA.TESTING_EPHEMERAL does not exist or not authorized.</code><br></br>

Author: ```suggestion
 <code>Object 'DATABASE.SCHEMA.METRIC_MODEL_TABLE' does not exist or not authorized.</code><br></br>
```
